src/azure-cli/azure/cli/command_modules/rdbms/_help.py

```diff
@@ -709,6 +709,31 @@
 short-summary: Show server key.
 """
 
+helps['mysql server ad-admin'] = """
+type: group
+short-summary: Manage a mysql server's Active Directory administrator.
+"""
+
+helps['mysql server ad-admin create'] = """
+type: command
+short-summary: Creates a new mysql server Active Directory administrator.
+"""
+
+helps['mysql server ad-admin delete'] = """
+type: command
+short-summary: Deletes an existing mysql server Active Directory Administrator.
+"""
+
+helps['mysql server ad-admin list'] = """
+type: command
+short-summary: Returns a list of mysql server Active Directory Administrators.
```

Comment:
Something like "List all Active Directory Administrators for mysql server" will be better.